documentation/internal/Overview-of-Navigation-&-URL-structure.md

```diff
@@ -359,10 +357,16 @@ Example: [https://balsamiq.cloud/scs7giw/ponj59g/r6C38](https://balsamiq.cloud/s
 
 ---
 
-## `/products/[product-id]`
+## `/products/[productId]`
```

Comment:
```suggestion
## `/product/[productId]`
```

To keep consistent with `application`  and `template`

Reply:
Fair call -- have also done the same to `notifications`.